src/pages/Merchant/Register/Register.js

```diff
@@ -98,71 +141,114 @@ function Register() {
+                onChange={(event) =>
+                  setValues({ ...values, email: event.target.value })
+                }
+                value={values.email}
+                error={errors?.email}
               />
+              <div className={styles["registerRightPanel-inputContainer"]}>
+                <InputControl
+                  password="true"
+                  placeholder="Enter password"
+                  label="Password"
+                  onChange={(event) =>
+                    setValues({ ...values, password: event.target.value })
+                  }
+                  value={values.password}
+                  error={errors?.password}
+                />
 
+                <InputControl
+                  placeholder={`Confirm Password`}
+                  label={`Confirm Password`}
+                  onChange={(event) =>
+                    setValues({ ...values, confirmpass: event.target.value })
+                  }
+                  value={values.confirmpass}
+                  error={errors?.confirmpass}
+                  password="true"
+                />
+              </div>
+              <p>
+                Already have an account.&nbsp;
+                <span
+                  className={styles["registerRightPanel_helper-text"]}
+                  onClick={changeURl}
+                >
+                  Login now
+                </span>
+              </p>
+              <Button type={`submit`}>Register</Button>
+            </div>
+            <div id="recaptcha"></div>
+          </form>
+        </div>
+      ) : (
+        <div className={styles["registerRightPanel_otp"]}>
+          <form onSubmit={handleOtpVerification} className={styles.otpForm}>
+            <div className={styles["registerRightPanel-mainBody"]}>
+              <h2>Verify OTP</h2>
               <InputControl
-                placeholder={`Confirm Password`}
-                label={`Confirm Password`}
-                ref={confirmpassRef}
-                error={errors?.confirmpass}
-                password="true"
+                label="OTP"
+                placeholder="Enter OTP"
+                maxLength={6}
+                onChange={(event) =>
+                  setValues({ ...values, otp: event.target.value })
+                }
+                value={values.otp}
+                error={errors?.otp}
               />
+              <p>
+                <span onClick={handleSubmit}>Resend OTP</span>
+              </p>
+              <Button type={`submit`}>Verify OTP</Button>
             </div>
-            <p>
-              Already have an account.&nbsp;
-              <span
-                className={styles["registerRightPanel_helper-text"]}
-                onClick={changeURl}
-              >
-                Login now
-              </span>
-            </p>
-            <Button type={`submit`}>Create Account</Button>
-          </div>
-        </form>
-      </div>
+          </form>
+        </div>
```

Comment:
this thing should have a different component right? as discussed

Reply:
ok